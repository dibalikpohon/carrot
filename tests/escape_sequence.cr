println("ABC\nDEF")
println("ABC\tDEF")
println("ABC\\DEF")
println("\"ABC\"")
println("\'ABC\'")
println("'ABC'")
