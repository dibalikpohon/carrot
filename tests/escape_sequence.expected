ABC
DEF
ABC	DEF
ABC\DEF
"ABC"
'ABC'
'ABC'
